cmd/argocd/commands/repo.go

```diff
@@ -215,15 +216,41 @@ func NewRepoRemoveCommand(clientOpts *argocdclient.ClientOptions) *cobra.Command
 				c.HelpFunc()(c, args)
 				os.Exit(1)
 			}
+			var numOfRepo = len(args)
+			var isConfirmAll bool = false
 			conn, repoIf := headless.NewClientOrDie(clientOpts, c).NewRepoClientOrDie()
 			defer io.Close(conn)
 			for _, repoURL := range args {
-				_, err := repoIf.DeleteRepository(ctx, &repositorypkg.RepoQuery{Repo: repoURL})
-				errors.CheckError(err)
-				fmt.Printf("Repository '%s' removed\n", repoURL)
+				var lowercaseAnswer string
+				if !noPrompt {
```

Comment:
if `prompt == yes` (true) then i think we don't need to loop repo to take user consent as user already provide the "yes" in the command

Reply:
Thanks for your review. I tried your suggestion, but there will be one more for loop and some degree of code duplication. Relatively speaking, this is more concise now, and it is done elsewhere, like: https://github.com/argoproj/argo-cd/blob/master/cmd/argocd/commands/app.go#L1144 . Are there other performance or other convenience considerations in your suggestion?